Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
```

Comment:
### [marcel](https://github.com/rails/marcel)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.3.3 → 1.0.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/rails/marcel/compare/v0.3.3...v1.0.1).

These are the first 10 commits:
- (491b2a7) [Don't require "pathname" stdlib to be required](https://github.com/rails/marcel/commit/491b2a73957d3cd046549439bd88f09a34b4204a)
- (0f2f676) [Merge pull request #15 from janko/fix-pathname-constant-erro…](https://github.com/rails/marcel/commit/0f2f6762e2264b2dab34be388bedea3abcc39e41)
- (1f2468f) [Update Readme - add references section](https://github.com/rails/marcel/commit/1f2468f307c243328298e86b124bf85a3ae1e7dc)
- (8b62388) [README: Use syntax highlighting](https://github.com/rails/marcel/commit/8b62388d5146592be22abf270ce5e7ce4660eb00)
- (08b08ab) [Gemspec: Allow Bundler 2](https://github.com/rails/marcel/commit/08b08abab886860c269493534b805a41266ebbe9)
- (1dea288) [Bump rack from 2.0.3 to 2.1.4](https://github.com/rails/marcel/commit/1dea28861fa3c86d3608e27a176dcf285fa4a392)
- (cce58f2) [Update rake requirement from ~> 10.0 to ~> 13.0](https://github.com/rails/marcel/commit/cce58f26c728980a2e1f9ba5fe0b11aedf43b283)
- (5040f48) [Use MiniMime for MIME type lookup by filename/extension](https://github.com/rails/marcel/commit/5040f4875e63b09455e9ef99e5943d561d773848)
- (ee26e6c) [Update rubies and bundler](https://github.com/rails/marcel/commit/ee26e6c4fa296590efc60b1a8b086136a040c3d7)
- (91de487) [Merge pull request #28 from eileencodes/upgrade-marcel-rubie…](https://github.com/rails/marcel/commit/91de48709b0d2a763164e80045c0c749139f4ba1)


</details>